invenio_vocabularies/contrib/subjects/datastreams.py

```diff
@@ -12,6 +12,15 @@
 from invenio_i18n import lazy_gettext as _
 
 from ...datastreams.writers import ServiceWriter
+from .euroscivoc.datastreams import (
+    VOCABULARIES_DATASTREAM_READERS as euroscivoc_readers,
+)
+from .euroscivoc.datastreams import (
+    VOCABULARIES_DATASTREAM_TRANSFORMERS as euroscivoc_transformers,
+)
+from .euroscivoc.datastreams import (
+    VOCABULARIES_DATASTREAM_WRITERS as euroscivoc_writers,
+)
```

Comment:
```suggestion
from .euroscivoc.datastreams import (
    VOCABULARIES_DATASTREAM_READERS as euroscivoc_readers,
    VOCABULARIES_DATASTREAM_TRANSFORMERS as euroscivoc_transformers,
    VOCABULARIES_DATASTREAM_WRITERS as euroscivoc_writers,
)
```

nit: this also works I think. The alternative is to do something like:

```suggestion
from .euroscivoc import datastreams as euroscivoc_datastreams

# ...further down

VOCABULARIES_DATASTREAM_READERS = {**mesh_readers, **euroscivoc_datastreams.VOCABULARIES_DATASTREAM_READERS}

...
```

I let you decide your favourite 💅, and apply also the MeSH imports